extensions/Worker.Extensions.ServiceBus/release_notes.md

```diff
@@ -4,8 +4,7 @@
 - My change description (#PR/#issue)
 -->
 
-### Microsoft.Azure.Functions.Worker.Extensions.ServiceBus 5.14.1
+### Microsoft.Azure.Functions.Worker.Extensions.ServiceBus 5.15.0
 
-- Fixed issue where deadlettering a message without specifying properties to modify could throw
-  an exception from out of proc extension.
-- Include underlying exception details in RpcException when a failure occurs.
+- Update `Microsoft.Azure.WebJobs.Extensions.ServiceBus` dependency to `5.13.4`
+- Update `Microsoft.Azure.Extensions` to `1.7.1`
```

Comment:
Why do we mention this transitive dependency specifically?

Reply:
Nevermind, I see this is for the isolated worker dependency.

Reply:
This is explicitly referenced in the csproj. I updated it as well as it is what brings in `Azure.Identity`.